docs/user-guide/creating-application/deployment-template.md

```diff
@@ -1,34 +1,31 @@
-# Devtron Charts
+# Deployment Template
 
-**Make sure Global Configuration > GitOps is configured before moving ahead.**
 
-A deployment configuration is a manifest for the application. It defines the runtime behavior of the application.
+A deployment configuration is a manifest of the application. It defines the runtime behavior of the application.
 
 Devtron includes deployment template for both default as well as custom charts created by a super admin.
 
-To configure a deployment chart for your application:
+To configure a deployment chart for your application, do the following steps:
 
 * Go to **Applications** and create a new application.
 * Go to **App Configuration** page and configure your application.
-* On the **Deployment Template** page, select the drop-down under **Chart type**.
+* On the **Base Deployment Template** page, select the drop-down under **Chart type**.
 
 You can select a chart in one of the following ways:
 
-## Select chart by Devtron
-
 1. [Rollout Deployment](deployment-template/rollout-deployment.md)
-2. [Job and Cronjob](deployment-template/job-and-cronjob.md)
+2. [Cronjob & Job](deployment-template/job-and-cronjob.md)
```

Comment:
Is it Cronjob & Job on the UI?

Reply:
It is existing as Cronjob & Job in UI, so I changed the same in doc also (but later I realised what if the UI is wrong or to be updated), so I reverted the changes. This has to be confirmed. Moreover, there is a new feature where a new deployment helm chart is to be added. So, we can have a look more in detail on all the charts including Cronjob and Job.